README.md

```diff
@@ -1,36 +1,34 @@
-Bazel rules for apko
-====================
+# Bazel rules for apko
```

Comment:
Aw, but the other H1/H2 style is so much more readable in source form

Reply:
It's auto-formatting :(